### PR TITLE
4255 - Migrate to vuejs v3

### PR DESCRIPTION
**Pull request by ericangeles**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**
Migrating from vue 2 to vue 3.

**Related github/jira issue (required)**:
Fixes https://github.com/infor-design/enterprise/issues/4255

**Steps necessary to review your pull request (required)**:
- Pull this branch
- Go to vuejs project folder `enterprise-wc-examples/vue-ids-wc`
- Run `npm install`
- Then `npm run serve`
- Go to http://localhost:8080/

### CODE REVIEW

**Comment by ericangeles** [comment]
Still WIP fyi

**Comment by ericangeles** [comment]
I guess it's all good now, @tmcconechy

File: vue-ids-wc/src/App.vue [hunk 1]
```diff
@@ -1,20 +1,19 @@
...
+  <IdsTagTestPage />
 </template>
 
 <script>
-import IdsTagTestPage from './components/Tag.vue';
+import IdsTagTestPage from "./components/Tag.vue";
+
 export default {
-  name: 'App',
+  name: "App",
```
Comment (tmcconechy) [inline vue-ids-wc/src/App.vue:9]:
```suggestion
  name: 'App',
```

File: vue-ids-wc/src/components/Tag.vue [hunk 2]
```diff
@@ -42,17 +38,17 @@ const data = [
 ];
 
 export default {
-  name: "TagTestPage",
+  name: "IdsTagTestPage",
```
Comment (tmcconechy) [inline vue-ids-wc/src/components/Tag.vue:41]: You have a mix of single and double quotes, generally we us single in JS and double in HTML

**Review by tmcconechy — COMMENTED** [review]
Looks good just a minor mix of single and double tags. Also i think you can remove the #App css styles or reduce them. They seem to be adding a lot of padding and changing the text color.

**Pull request merged**